invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessLinks/LinksSearchItem.js

```diff
@@ -66,67 +78,82 @@ export const LinksSearchItem = ({ result, record, fetchData }) => {
 
   return (
     <Table.Row key={result.id}>
-      <Table.Cell width={3} data-label="Link title">
-        {isEmpty(result.description)
-          ? "-"
-          : _truncate(result.description, { length: 60 })}
-      </Table.Cell>
-      <Table.Cell width={3} data-label="Created">
-        {timestampToRelativeTime(result.created_at)}
-      </Table.Cell>
-      <Table.Cell width={3} data-label="Expires at">
-        {isEmpty(result.expires_at)
-          ? i18next.t("Never")
-          : `${timestampToRelativeTime(result.expires_at)} (${result.expires_at})`}
-      </Table.Cell>
-      <Table.Cell width={3} data-label="Access">
-        <AccessDropdown
-          updateEndpoint={`${record.links.access_links}/${result.id}`}
-          dropdownOptions={dropdownOptionsGenerator(dropdownOptions)}
-          result={result}
-        />
-      </Table.Cell>
-      <Table.Cell width={4}>
-        <Button
-          loading={loading}
-          disabled={loading}
-          onClick={handleDelete}
-          icon
-          labelPosition="left"
+      {error ? (
```

Comment:
minor: could we discuss not using ternary operators with big blocks of code? it is a bit unreadable

Reply:
I put the message inside the row because it's an error that occurred on deleting a specific secret link. Do you mean we should put the error message on top of the modal?

Reply:
ok I see, that was not obvious from the code, thanks for explaining.
What about ternary operators?

Reply:
Replaced ternany operators with 
```
{error && <ErrorMessage />}
{!error && <... />}
```
I agree looks more readable now!